cypress/e2e/smoke/create_tx.cy.js

```diff
@@ -35,89 +40,83 @@ describe('Queue a transaction on 1/N', () => {
   })
 
   it('should create a queued transaction', () => {
-    // Spy the /estimations request
-    cy.intercept('POST', '/**/multisig-transactions/estimations').as('estimations')
-
     // Alias for New transaction modal
     cy.contains('h2', 'Review transaction').parents('div').as('modal')
 
     // Wait for /estimations response
-    cy.wait('@estimations', { timeout: 30_000 })
-
-    // Estimation is loaded
-    cy.get('button[type="submit"]').should('not.be.disabled')
-
-    // Gets the recommended nonce
-    cy.contains('Signing the transaction with nonce').should(($div) => {
-      // get the number in the string
-      recommendedNonce = $div.text().match(/\d+$/)[0]
+    cy.intercept('POST', '/**/multisig-transactions/estimations').then(() => {
```

Comment:
For how long does the test try to intercept the request? the default timeouts are usually 5sec so it might be worth to keep the 30sec timeout that we had

Reply:
if it waits for the request, all good. I will resolve then